crates/aggchain-proof-service/src/service.rs

```diff
@@ -90,6 +100,17 @@ impl AggchainProofService {
         .map_err(Error::AlloyProviderInitializationFailed)?;
         let l1_rpc_client = Arc::new(client);
 
+        // TODO: Remplace this DUMMY vkey by the reel one
+        let vkey = [0u32; 8];
+        let chain_l1_rpc = L1Rpc {};
```

Comment:
You have this in the `config.proposer_service.l1_rpc_endpoint`. Probably best to make the config for the `CustomChainDataBuilderService`.